pdns/recursordist/test-syncres_cc5.cc

```diff
@@ -1176,15 +1176,15 @@ BOOST_AUTO_TEST_CASE(test_dnssec_validation_nsec3_wildcard) {
   BOOST_CHECK_EQUAL(res, RCode::NoError);
   BOOST_CHECK_EQUAL(sr->getValidationState(), Secure);
   BOOST_REQUIRE_EQUAL(ret.size(), 6);
-  BOOST_CHECK_EQUAL(queriesCount, 10);
+  BOOST_CHECK_EQUAL(queriesCount, 9);
```

Comment:
Did you check why the number of queries change? I'm pretty sure this is not normal, and at first glance it is caused by a `NODATA` for `sub.powerdns.com|DS` somehow being leading to `www.sub.powerdns.com: Entire name 'sub.powerdns.com' and below,is negatively cached via 'powerdns.com' for another 300 seconds`.